docs/plugins.md

```diff
@@ -87,7 +87,7 @@ These plugins apply transformations to your code.
 - [es2015-block-scoping](/docs/plugins/transform-es2015-block-scoping/)
 - [es2015-classes](/docs/plugins/transform-es2015-classes/)
 - [es2015-computed-properties](/docs/plugins/transform-es2015-computed-properties/)
-- [es2015-constants](/docs/plugins/check-es2015-constants/)
```

Comment:
No worries @andimarek!

I agree it is semi-out of place in a section titled "Transform Plugins", but I think it's a fine exception for now (vs. creating a whole new "check" section).

Do you mind reopening and renaming it to [check-es2015-constants] to help others who might be confused what this links to?